tools/create_tensorflow_warmup.py

```diff
@@ -0,0 +1,71 @@
+import os
+import tensorflow as tf
+
+from os import path
+from absl import app, flags
+from absl.flags import FLAGS
+from tensorflow_serving.apis import model_pb2, predict_pb2, prediction_log_pb2
+
+from yolov3_tf2.dataset import preprocess_image, load_tfrecord_dataset
+
+
+flags.DEFINE_string('dataset', None, 'path to the dataset, labels will be ignored')
```

Comment:
Estou tentando organizar as FLAGS nossas no padrão abaixo:
flags.DEFINE_string('classes', './data/coco.names',
                    'path to classes file')
flags.DEFINE_string('checkpoints', './out',
                    'folder to save the checkpoints to')

flags.DEFINE_integer('size', 416,
                     'image size')

flags.DEFINE_float('learning_rate', 1e-3,
                   'learning rate')
 
 flags.mark.......

Reply:
Ok, separei e alinhei as flags em 5f677274bc82b8b87208ac51d6328170812d7c4a